docs.html

```diff
@@ -1930,7 +1930,7 @@
 									</li>
 								</ol>
 							</dd>
-							<dt>Menus.addMenuItem(menuId, menuItemTitle, menuItemURL, [menuItemUIRoute], [isPublic], [roles]);</dt>
+							<dt>Menus.addMenuItem(menuId, menuItemTitle, menuItemType, menuItemURL, [menuItemUIRoute], [isPublic], [roles], [position]);</dt>
```

Comment:
Shouldn't be `Menus.addMenuItem(menuId, menuItemTitle, menuItemURL, menuItemType, [menuItemUIRoute], [isPublic], [roles], [position]);` ?